qtpy/tests/test_qtcharts.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import absolute_import
+
+import pytest
+from qtpy import PYQT5, PYSIDE2
+
+
+@pytest.mark.skipif(not (PYQT5 or PYSIDE2), reason="Only available in Qt5 bindings")
```

Comment:
Please change the condition to `not PYSIDE2` because it's not easy to test this for PyQt5.